### DISCUSSION TITLE
looking for medication module that also consideres adverse drug reactions

### DISCUSSION THREAD

**Discussion by julia-dyck** [synthetichealth/synthea#1447] [Q&A]
Hi, I am looking for a medication module that also considers the occurrence of adverse drug reactions (ADRs).

To be more specific: I need synthetic data of patients, who take a specific drug over some time and do or do not get a specific adverse reaction diagnosed after they started the medication.
The goal is to demonstrate the usage of a signal detection tool developed to identify time-dependent correlations between drug intake and symptoms suspected to be an ADR (have a look at an existing approach with data example described [here](https://link.springer.com/content/pdf/10.1007/s40264-013-0061-7.pdf) to get an idea).

Are there medication modules that have the occurrence of adverse events implemented already?
I'd be happy about any suggestions!

**Comment by jawalonoski** [comment]
@julia-dyck, Synthea does not contain adverse drug reactions, if my memory serves.

Typically, most disease modules feature one or two different care paths. If medication is involved, sometimes the medications are changed entirely to a different treatment, or sometimes they are escalated (e.g., increased dosage, or combined with another drug). I don't recall specific adverse events being recorded any of those instances.

The system is capable of producing that data (in one form or another), but those states/data would have to be added to the modules, requiring editing. So you probably won't find the information you're looking for out of the box.

**Comment by annayudovin** [comment]
If you are still interested in pursuing this, (and don't feel like diving that deeply into Synthea, given a lack of stock modules that model ADRs), I would be happy to customize a module of your choice with any ADRs you want to specify - with any frequency, delay, and (possibly) additional dependencies (like a co-morbidity, or additional medications). Since I already have an instance of Synthea set up and running, I can even generate some datasets for you, as long as you can tell me what you are looking for.

I am actually very curious about how "realistic" the results will be, and this is topical to something I'm working on anyway, so it is ABSOLUTELY worth my time to try this out.
